src/libraries/Microsoft.PowerFx.Core/Public/Values/StringValue.cs

```diff
@@ -28,6 +28,12 @@ internal StringValue ToLower()
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
+            if (settings.UseCompactRepresentation)
+            {
+                sb.Append($"\"{Value}\"");
+                return;
+            }
```

Comment:
Can we avoid this?

Reply:
+1. The compact representation should be as much as possible a valid PFx expression, and that may not be the case.

Reply:
I was under the impression that it was the very opposite. The compact representation should not be as much as possible a valid PFx expression, although for many cases it is. Take what the serializer produces for an empty table, an empty record and a BlankValue value.
Lets take the following test case as an example:
```
>> "ab""cd"
"ab"cd"
```
Shouldn't we compare the expected value against Eval's result? It sounds like we are testing the serializer and not Eval.

Reply:
Settled offline. Please check **StringInterpolate.txt** for the changes related to this comment.